src/vmm/src/builder.rs

```diff
@@ -142,51 +145,6 @@ impl std::convert::From<linux_loader::cmdline::Error> for StartMicrovmError {
     }
 }
 
-// Wrapper over io::Stdin that implements `Serial::ReadableFd` and `vmm::VmmEventsObserver`.
```

Comment:
The wrapper introduces more code in its definition than it saves in de-duplication. I also find the additional re-direction to make the code less readable.

Reply:
the reason for removing the wrapper makes sense, I just wanted to have the reason mentioned somewhere so that it is visible for future audience why this was done.
nit: BTW, replacing the wrapper would be a great candidate to be added in a separate commit.
something to make a note of for future code changes as well.

Reply:
Removing the wrapper also relates to in-lining `on_vmm_boot` and `on_vmm_stop` it would be a bit of an involved rebase to split it out now. Would it be sufficient to add the reason to the commit message?

Reply:
Added the comment on the commit message.